pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -79,6 +79,18 @@ if [ "$NIX_ENFORCE_PURITY" = 1 -a -n "$NIX_STORE" ]; then
     params=("${rest[@]}")
 fi
 
+if test -n "@libcxx@"; then
+    NIX_CFLAGS_COMPILE="$NIX_CFLAGS_COMPILE -isystem@libcxx@/include/c++/v1"
```

Comment:
Yes, let's just put the `NIX_SKIP_CXX[ABI]` variables into the setup hook as well. @gridaphobe If that's obvious how you should do it please do, otherwise I'll take a look when I get a chance.